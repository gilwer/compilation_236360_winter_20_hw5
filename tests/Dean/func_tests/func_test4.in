

void f(bool cond){
	if(cond){
		print("boy");
		f(false);
	}else{
		print("girl");
	}
	
}	

bool g(bool gen){
	if(gen){
		print("M "); 
		return true;
	}
	else{
		print("F ");
		return false;
	}
}


void main(){
	print("apple ");
	f(g(true));
	f(g(false));
}