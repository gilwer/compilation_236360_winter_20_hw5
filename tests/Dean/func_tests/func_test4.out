apple 
M 
boy
girl
F 
girl
